use-cases/GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,238 @@
+# Using Intel GPU device with Kata Containers
+
+Support for using Intel GPU devices with Kata Containers has been added in  Kata Containers version 1.3.0.
```

Comment:
s/added in/in Kata Containers since version/

Reply:
done